doctr/datasets/classification/base.py

```diff
@@ -30,7 +30,7 @@ def synthesize_char_img(char: str, size: int = 32, font_family: Optional[str] =
 
     # Draw the character
     if font_family is None:
-        font = ImageFont.load_default()
+        font = ImageFont.truetype("FreeMono.ttf", size)
```

Comment:
I disagree, with this behaviour, 2 out of our 3 main supported OS will raise an error. FreeMono is Ubuntu-specific.
As mentioned in #416 & #417, we'd better detect the OS family and assign predefined default fonts to each one of them